template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/databricks.yml.tmpl

```diff
@@ -15,18 +15,18 @@ variables:
 include:
```

Comment:
As the comment suggested, let's just use `- ./resources/*`. Maybe merge the comments into one about what's included in the `resources` directory.

Reply:
Do we want to include feature engineering workflow by default (I guess it is fine as it is not being used).

Reply:
The feature engineering workflow will not present in the directory if it's not used.